dev/strongpoc/serializers.py

```diff
@@ -22,7 +22,7 @@ class Meta:
 
 class PointOfContactSerializer(serializers.ModelSerializer):
     # team = serializers.StringRelatedField()
-    team = TeamSerializer()
+    # team = TeamSerializer()
```

Comment:
@jathanism Do you remember why we were doing this?  It caused problems where the POST wanted an entire dict for team and wanted to try to create the team record as well as the POC record.  So I commented it back out.  So far, I'm not seeing any issues with doing that.